### PR TITLE
Added info in README.md [ci skip]

### PR DESCRIPTION
**Pull request by sergeysviridenko**
Hello!

* Type: documentation
* Link to issue: #1

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines][:contrib:]
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR

Small description of change:
Added information about permission on the directory

Thanks

[:contrib:]: https://github.com/phalcon/phalcon-devtools/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,11 @@ cd vokuro
 php composer.phar install
 ```
 
+### After instalation should set right on the directories
```
Comment (niden) [inline README.md]: What do you mean by that? `set correct permissions` ?

Comment (sergeysviridenko) [inline README.md]: I agree - it not so correct. I mean folder should be writable.

Comment (niden) [inline README.md]: OK so here is a suggestion:

`### After the installation, please ensure that the following folders have write permissions set`

**Comment by sergeyklay** [comment]
Thank you

**Comment by sergeysviridenko** [comment]
Thank you.

**Pull request merged**